### ISSUE TITLE
Delete all built-in Universal Windows Platform applications

### ISSUE THREAD

**Issue by PF94** [Craftplacer/TweakUtility#20]

**Comment by Craftplacer** [comment]
I would consider making this into an advanced feature, think of it like, you can choose to either remove the UWP apps that make your PC into a phone, or remove all of them, and then including the option of restoring Windows 7 versions of the accessories.

**Comment by AlienFuture** [comment]
I mean these apps ex.: 3D Viewer, Movies & TV, Groove Music etc. etc. Maybe yea we can make it via checkboxes with uninstall button at the end.

**Comment by Craftplacer** [comment]
I was still looking around for calling Windows functions to remove them instead of relying on PowerShell. Couldn't find any really, the least I could find was a .NET UWP library but can't track it down where it is.

**Comment by AlienFuture** [comment]
I had a powershell script for that, i will search back later.

**Comment by Craftplacer** [comment]
Yeah Get-AppXPackages etc.

**Comment by PF94** [comment]
does "get-appxpackages" exist in Windows 8.1, and 8.0, and the 8.0 betas?

**Comment by AlienFuture** [comment]
UWP comed with Windows 10. Windows 8 uses Metro apps

**Comment by PF94** [comment]
UWP was a expansion of the way how Metro apps worked in 8.1.

**Comment by PF94** [comment]
> including the option of restoring Windows 7 versions of the accessories

WinAero has that, But please @Craftplacer don't use their installers, Just make alternative ones.

Also if someone wants to restore the games. They should have the option of "Windows 7 Games" and "Windows XP Games"

**Comment by AlienFuture** [comment]
ill search about it

**Comment by Craftplacer** [comment]
Nah, stealing directly from WinAero won't be an option, this would be just being lazy. We can figure out how stuff works.

**Comment by AlienFuture** [comment]
~~Download winareo win7 games installer into TU directly~~
just joke lol
